openlibrary/components/LibraryExplorer.vue

```diff
@@ -139,7 +161,7 @@ export default {
   font-family: "Avenir", Helvetica, Arial, sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  color: #2c3e50;
+  color: rgba(0, 0, 0, .7);
```

Comment:
Q: Is there any way to move these classes outside of vue so we can leverage our existing less and avoid the possibility of introducing new colors?